opencti-platform/opencti-front/src/private/components/common/files/import_files/ImportFilesDialog.tsx

```
@@ -216,15 +283,121 @@ const ImportFilesDialog = ({ open, handleClose, entityId }: ImportFilesDialogPro
     });
   };
 
+  const onSubmit: FormikConfig<OptionsFormValues>['onSubmit'] = async (values, { setErrors }) => {
+    setUploadStatus('uploading');
+    const selectedEntityId = entityId ?? (values.associatedEntity?.value || undefined);
+    const fileMarkingIds = values.fileMarkings.map(({ value }) => value);
+
+    const { validationMode, name } = values;
+    if (validationMode === 'workbench') {
+      importFiles({ selectedEntityId, fileMarkingIds, validationMode }, setErrors);
+    } else {
+      const newDraftId = await createDraft(name);
```

If draft creation fails, I think we should handle things differently here. 
Right now, even if draft creation fails, we still try to import files. We should raise an error if draft creation fails, and not proceed to trying to import the files